python/feathub/feature_tables/feature_table.py

```diff
@@ -74,6 +74,20 @@ def __init__(
         self.properties = properties
         self.schema = schema
 
+        if schema:
```

Comment:
Is it safer if we explicitly check `if schema is not None`? Same for the other None check.